test/eventrouter.sh

```diff
@@ -27,8 +33,31 @@ else
     esopssvc=$( get_es_svc es-ops )
     esopssvc=${esopssvc:-$essvc}
 
+    # Make sure there's no MUX
+    oc set env ds/logging-fluentd MUX_CLIENT_MODE- 2>&1 | artifact_out
+    oc label node --all logging-infra-fluentd-
+    oc label node --all logging-infra-fluentd=true
```

Comment:
the `oc set env` has the effect of restarting fluentd - if that is sufficient in this case, you can get rid of the `oc label node` and add something like this, to wait for fluentd to be up and running.  That is, the usual idiom is  - label node with `logging-infra-fluentd-` to shut off fluentd, then wait for the daemonset numberReady to be 0.  Then, edit the fluentd ds, configmap, whatever changes are needed to fluentd.  Then, relabel the node, and wait for a fluentd pod to be running:
```
    # undeploy fluentd
    oc label node --all logging-infra-fluentd- 2>&1 | artifact_out
    os::cmd::try_until_text "oc get daemonset logging-fluentd -o jsonpath='{ .status.numberReady }'" "0" $FLUENTD_WAIT_TIME
    # oc set env, edit ds, whatever
    oc label node --all logging-infra-fluentd=true 2>&1 | artifact_out
    os::cmd::try_until_text "oc get pods -l component=fluentd" "^logging-fluentd-.* Running "
```